CMakeLists.txt

```diff
@@ -167,6 +171,8 @@ option(DCA_WITH_TESTS_EXTENSIVE "Build DCA++'s extensive tests." OFF)
 option(DCA_WITH_TESTS_PERFORMANCE "Build DCA++'s performance tests. (Only in Release mode.)" OFF)
 option(DCA_WITH_TESTS_STOCHASTIC  "Build DCA++'s stochastic tests." OFF)
 
+set(DCA_TEST_GPU_COUNT "${SYSTEM_GPU_COUNT}" CACHE INTEGER "Number of GPUs available on one node for one test.")
```

Comment:
Where is the number of GPUs / node used?

Reply:
I have now pointed it out for you.

Reply:
If there is only one GPU on the node the ringG test is not built.

Reply:
this is not a necessary check as the ringG has the exact same requirement of the other DCA tests: 1 GPU / rank. Plus cuda aware MPI if available.

Reply:
See below, if a test is primarily to check a distributed algorithm and the communicaiton calls in it.  Then it doesn't "pass" if its run on one rank.